src/views/Static/Community/Community.vue

```diff
@@ -308,7 +308,7 @@
           >
             <v-card-title>
               <h4 style="word-break: initial">
-                <a :href="tool.homepage">
+                <a :href="tool.homepage" :id="tool.id">
```

Comment:
```suggestion
                <a 
                  :id="tool.id"
                  :href="tool.homepage" 
                >
```

Reply:
Codacy isn't happy about those two parameters being on a single line, and :href being first, so this suggestion moves them accordingly.